src/components/admin/EventsPanel/EventsPanel.tsx

```diff
@@ -39,7 +41,7 @@ export const EventsPanel: React.FC<EventsPanelProps> = ({
     [spaces]
   );
 
-  const hasVenueEvents = events?.length !== 0;
+  const hasSpaceEvents = events?.length !== 0;
```

Comment:
Is this OK? 
If `events?.length` returns `undefined` the result will be `true`.

There are multiple ways to solve this, two examples are: `(events?.length ?? 0) !== 0` or simpler ` !! events?.length `

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2941/commits/45c547cc49b25a3a7391b6468e28379ec01dd5b2